### DISCUSSION TITLE
How make TST reopen previously opened tabs when browser starts - have enabled in FireFox settings

### DISCUSSION THREAD

**Discussion by infonut** [piroor/treestyletab#3402] [Q&A]
Hi,
I swear I remember checking a box to close all tabs on browser closing and I now want to do the opposite of this behavior and have them reopen when FFox starts. I can't find the box I checked. (Or maybe there is no box and I am misremembering?)

FYI 1: I have this checked in FFox Settings:
Startup
Open previous windows and tabs

FYI 2 : I have these 2 additional TST extensions (I have looked in them as well and can't find it there either)

Auto Tab Discard
--I do NOT have this box checked:  | Discard all unpinned tabs on a browser or extension startup
--I do NOT have this box checked:  | Discard all pinned tabs on a browser or extension startup

Tab Unloader for Tree Style Tab
--I don't see any tabs that I have checked that relate to reopening/not reopening tabs on startup

I think this is a simple check mark somewhere, I just can't find it!

Thanks in advance for your help

**Comment by piroor** [comment]
TST has no such option and simply respects Firefox's "Open previous windows and tabs" behavior. TST is designed to restore tree structure of tabs when tabs are restored by Firefox itself.
